frameworks/PHP/fomo/.env

```diff
@@ -0,0 +1,22 @@
+APP_NAME=Fomo
+APP_TIMEZONE=UTC
+APP_WORKER_COUNT=8
```

Comment:
Delete this line, so the server will adapt to the computer CPUs.

In server.php
```
'additional' => [
        'worker_num' => env('APP_WORKER_COUNT' , cpuCount() * 2) ,
```

:hand: @n0nag0n